src/common/computeReadiumCssJsonMessage.ts

```diff
@@ -39,7 +39,7 @@ export const computeReadiumCssJsonMessage = (settings: ReaderConfig): IEventPayl
 
         lineHeight: settings.lineHeight,
 
-        night: settings.night,
+        // night: settings.night,
```

Comment:
This is fundamentally altering how ReadiumCSS works, it is essentially an API breaking change. The solution is to inject specific foreground / background colours when the theme is neither "sepia" nor "night" (in other words, when in "neutral" mode, force `--RS__backgroundColor` and `--RS__textColor`). I will write the necessary plumbing in the navigator so that Thorium can push the colors into ReadiumCSS.

Reply:
fixed here and in navigator